FUNDABLES.md

```diff
@@ -260,6 +260,17 @@ need funding for user experience work, development, testing,
 infrastructure, potentially platform services (e.g., SMS), and community
 outreach.
 
+### Productionize Malware Detection
+PSF did an [RfP](https://github.com/python/request-for/blob/master/2019-Q4-PyPI/RFP.md) for some security work on [Warehouse](https://github.com/pypa/warehouse) in late 2019, but the goals for [Milestone 2](https://github.com/python/request-for/blob/master/2019-Q4-PyPI/RFP.md#milestone-2---systems-for-automated-detection-of-malicious-uploads) were more ambitious than funding allowed for. The malware detection system is currently in limbo: an interesting prototype with limited practical impact because of the astounding number of false-positives. Here is a sampling of some of the open issues:
```

Comment:
We assume that a person reading this page doesn't necessarily have a lot of context on current problems and why it's important to solve them. So here's a suggested change:

```suggestion
Since anyone can upload a package to PyPI, malicious users might upload malware, which would then harm users. To mitigate this risk, PSF [previously obtained funding](https://github.com/python/request-for/blob/master/2019-Q4-PyPI/RFP.md) to add some malware detection [Warehouse](https://github.com/pypa/warehouse) in late 2019, but the goals for [the relevant milestone](https://github.com/python/request-for/blob/master/2019-Q4-PyPI/RFP.md#milestone-2---systems-for-automated-detection-of-malicious-uploads) were more ambitious than funding allowed for. The malware detection system is currently in limbo: an interesting prototype with limited practical impact because of the astounding number of false-positives. To protect users from malware, we still need to:
```